examples/nlp/notebooks/biobert/biobert_ner.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell.\n",
+    "# !pip install wget\n",
+    "# !pip install git+https://github.com/NVIDIA/apex.git\n",
+    "# !pip install nemo_toolkit[nlp]\n",
+    "# !pip install unidecode\n",
+    "import os\n",
+    "import nemo\n",
+    "import nemo.collections.nlp as nemo_nlp\n",
+    "import numpy as np\n",
+    "import time\n",
+    "import errno\n",
+    "\n",
+    "from nemo.backends.pytorch.common.losses import CrossEntropyLossNM\n",
+    "from nemo.collections.nlp.nm.data_layers import BertTokenClassificationDataLayer\n",
+    "from nemo.collections.nlp.nm.trainables import TokenClassifier\n",
+    "from nemo.collections.nlp.callbacks.token_classification_callback import eval_epochs_done_callback, eval_iter_callback\n",
+    "from nemo.utils.lr_policies import get_lr_policy\n",
+    "from nemo import logging"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "BioBERT has the same network architecture as the original BERT, but instead of Wikipedia and BookCorpus it is pretrained on PubMed, a large biomedical text corpus, which achieves better performance in biomedical downstream tasks, such as question answering(QA), named entity recognition(NER) and relationship extraction(RE). This model was trained for 1M steps. For more information please refer to the original paper https://academic.oup.com/bioinformatics/article/36/4/1234/5566506.  For details about BERT please refer to https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedfornemo.\n",
+    "\n",
+    "\n",
+    "In this notebook we're going to showcase how to train BioBERT on a biomedical named entity recognition (NER) dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Download model checkpoint\n",
+    "Download BioBert/BioMegatron checkpoints from  NGC: https://ngc.nvidia.com/catalog/models and put the encoder weights \n",
+    "at `./checkpoints/biobert/BERT.pt` or `./checkpoints/biomegatron/BERT.pt` and the model configuration file at `./checkpoints/biobert/bert_config.json` or `./checkpoints/biomegatron/bert_config.json`."
```

Comment:
Could you please add biomegatron bert in here: https://github.com/NVIDIA/NeMo/blob/master/nemo/collections/nlp/nm/trainables/common/megatron/megatron_utils.py#L37
so that the checkpoint is downloaded automatically, maybe name it smth like biomegatron-bert-345m-cased